MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -834,6 +900,35 @@ namespace WPEFramework {
 	    return network_available;
 	}
 
+        bool MaintenanceManager::checkDeviceInitializationContextUpdate() {
```

Comment:
This function must not do the subscription. It must be done during MM initialization